drivers/sensor/st/lis2dw12/lis2dw12.h

```diff
@@ -58,6 +58,11 @@
 #define LIS2DW12_SHIFT_PM1		4
 #define LIS2DW12_SHIFT_PMOTHER		2
 
+/* shift value for 12 bit resolution */
+#define LIS2DW12_SHIFT_TEMP		4
+/* Temperature 12 bit scale factor */
+#define LIS2DW12_TEMP_SCALE_FACTOR		62500
```

Comment:
As we said I think it would be better to mention here that it is ```1000000/16``` because uC, so that one could relate to what it is written on DS